tests/framework/e2e/etcd_process.go

```diff
@@ -72,6 +73,7 @@ type EtcdServerProcess struct {
 	cfg        *EtcdServerProcessConfig
 	proc       *expect.ExpectProcess
 	proxy      proxy.Server
+	httpProxy  proxy.Server
```

Comment:
Don't know if I like the name `httpProxy`. Preferred the previous naming.
Why not have `reverseProxy` and `forwardProxy`?

Reply:
I will make the naming consistent throughout the PR!

Reply:
The naming convention is now aligned! 
Necessary blackhole-related failpoint changes are also made!